packages/@ember/-internals/glimmer/tests/integration/components/render-component-test.ts

```diff
@@ -0,0 +1,591 @@
+import {
+  AbstractStrictTestCase,
+  assertClassicComponentElement,
+  assertHTML,
+  buildOwner,
+  clickElement,
+  defComponent,
+  defineComponent,
```

Comment:
what's the difference between defComponent and defineComponent? 🙈

Reply:
idk -- this code was copied from a prior implementation attempt

It's ultimately not important right now, but I'm happy to rename this util in a separate PR :tada: